Assets/MRTK/Providers/Oculus/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef

```diff
@@ -2,9 +2,11 @@
     "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus",
     "references": [
         "Microsoft.MixedReality.Toolkit",
-        "Microsoft.MixedReality.Toolkit.Providers.XRSDK"
+        "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+        "Microsoft.MixedReality.Toolkit.XRSDK.MRTKQuest.Utilities",
```

Comment:
Nit: Thoughts on just going with OculusQuest or Quest or Oculus instead of MRTKQuest? I think when it was prefixed with prvncher it made more sense to have MRTKQuest (because then you knew what it was for)